_data/sites.json

```diff
@@ -1845,6 +1845,17 @@
         ]
     },
 
+    {
+        "name": "BorgBase",
+        "url": "https://www.borgbase.com/account?tab=5",
+        "difficulty": "easy",
+        "notes": "Remove all your backup repositories, then in the account tab click on profile and on the bottom of the page you'll find a button to remove your account.",
+        "email": "hello@borgbase.com",
```

Comment:
We only provide an e-mail when the user needs it to request the account deletion, but that doesn't seem to be the case from the notes you added. Can you please confirm?

Reply:
Yes. Good to know.